orocos_kdl/src/framevel.inl

```diff
@@ -317,7 +352,7 @@ void VectorVel::ReverseSign() {
 }
 doubleVel VectorVel::Norm() const {
     double n = p.Norm();
-    return doubleVel(n,dot(p,v)/n);
+    return doubleVel(n,n ? dot(p,v)/n : 0); // Setting norm of p to 0 in case norm of v is 0
```

Comment:
Comparing against exactly zero here, it might help to compare against `eps` if you want to prevent `Inf`, but in that case you might want to set n to zero as well?

Reply:
Will look into this.

Reply:
@smits please check https://github.com/orocos/orocos_kinematics_dynamics/pull/229/commits/a6fbd38c509ed7d00db0110fb0bf69afa879ad24 for my fix. The default argument has also been added to PyKDL